src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,123 @@
+package duke;
+
+import functions.*;
```

Comment:
You may want to list out all your imported classes explicitly instead of using wildcard symbol. This makes it easier for fellow developers to track the classes that have been imported into the class.